x-pack/legacy/plugins/canvas/public/components/expression_input/expression_language.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as monacoEditor from 'monaco-editor/esm/vs/editor/editor.api';
+
+interface Language extends monacoEditor.languages.IMonarchLanguage {
+  keywords: string[];
+  symbols: RegExp;
+  escapes: RegExp;
+  digits: RegExp;
+  boolean: ['true', 'false'];
+}
+
+export const language: Language = {
```

Comment:
Is this being used somewhere?

Reply:
Good catch! I'll remove this file. It's useless now that we have the `monaco_language_def.ts`